spring-boot-project/spring-boot/src/test/java/org/springframework/boot/context/properties/ConfigurationPropertiesBindExceptionTests.java

```diff
@@ -36,6 +36,7 @@ void createFromBeanHasDetails() {
 		ApplicationContext applicationContext = new AnnotationConfigApplicationContext(Example.class);
 		ConfigurationPropertiesBean bean = ConfigurationPropertiesBean.get(applicationContext,
 				applicationContext.getBean(Example.class), "example");
+		assertThat(bean).isNotNull();
```

Comment:
As above. Please revert. Additionally, `getBean(Class)` will never return `null`, throwing a `NoSuchBeanDefinitionException` instead.

Reply:
`ConfigurationPropertiesBean.get()` is `@Nullable`

Reply:
Sorry, I misread the diff. Regardless, the same reasoning as above still applies. I don't think null checks such as this in tests are beneficial.